packages/apps/frontera/src/routes/src/components/CommandMenu/commands/organization/ChangeRelationship.tsx

```diff
@@ -93,7 +93,23 @@ export const ChangeRelationship = observer(() => {
 
   return (
     <Command label='Change Relationship'>
-      <CommandInput label={label} placeholder='Change relationship...' />
+      <CommandInput
+        label={label}
+        placeholder='Change relationship...'
+        onKeyDownCapture={(e) => {
+          if (e.key === ' ') {
```

Comment:
what is the purpose of that?

Reply:
to not trigger another action since we have binded an action on space